scripts/onload.js

```diff
@@ -133,8 +133,10 @@ map.on('load', function() {
 			.setLngLat(e.lngLat)
 			.setHTML(fillpopup(e.features[0].properties))
 			.addTo(map);
+/* uncomment this block to restore the functionality that has clicks on points filtering the chart as well.
 		chartData.districtName = e.features[0].properties.NAME;
 		chartData.title = 'Charter schools located within ' + e.features[0].properties.NAME;
+*/
 		redrawChart();
```

Comment:
I left the `redrawChart();` call in because redrawing the chart _without changing any parameters_ has very little cost and can fix occasional glitches from resizing the browser window.

Reply:
Wow this was a simple change! Makes sense to retain redrawChart();

I noticed a weird glitch regarding disappearing drop-downs that I will email you about separately.